accounts-db/src/hardened_unpack.rs

```diff
@@ -339,7 +339,13 @@ pub fn streaming_unpack_snapshot<A: Read>(
         |_, _| {},
         |entry_path_buf| {
             if entry_path_buf.is_file() {
-                sender.send(entry_path_buf).unwrap();
+                let result = sender.send(entry_path_buf);
+                if let Err(err) = result {
+                    panic!(
+                        "failed to send path '{}' from unpacker to rebuilder: {err}",
+                        err.0.display(),
```

Comment:
nit: `entry_path_buf` instead of `err.0.display` right ?

Reply:
Oh nice, was not aware. And it looks like the `Display` impl for `SendError` does NOT print the `T` so we will not be double printing out the `T`.
https://docs.rs/crossbeam-channel/0.5.11/src/crossbeam_channel/err.rs.html#124-128

nit: Thoughts on using `into_inner()` over `.0` ? I doubt crossbeam would be adding additional items (and reordering them) at this point, but I feel like `into_inner()` is slighter more expressive
https://docs.rs/crossbeam/latest/crossbeam/channel/struct.SendError.html#method.into_inner

Reply:
Hmm yeah, I guess since we're inside a `let Err(err) = ...` block, we can't call `into_innter()` as that would move it. So, we'd have to clone; granted this isn't a big deal since the clone would be in the same line that we're panicking, but I think we can leave the `.0` too